api/src/org/labkey/api/reader/TabLoader.java

```diff
@@ -82,11 +81,11 @@ public DataLoader createLoader(File file, boolean hasColumnHeaders, Container mv
             return new TabLoader(file, hasColumnHeaders, mvIndicatorContainer);
         }
 
-        /** A DataLoader created with this constructor does NOT close the reader */
+        /** A DataLoader created with this constructor does NOT close the InputStream */
```

Comment:
I've always found this guidance a little confusing. Does this seem accurate and clearer?
```suggestion
        /** This constructor does NOT close the InputStream. Call close(), in a try-with-resources, if appropriate, to ensure resources are released.  */
```

Reply:
Sure. The guidance is a bit self-evident at this point, since `TabLoader` is `AutoCloseable`. But doesn't hurt to be explicit.